api/src/main/java/com/cloud/agent/api/to/LoadBalancerTO.java

```diff
@@ -179,6 +182,21 @@ public boolean isInline() {
         return inline;
     }
 
+    public LoadBalancerConfigTO[] getLbConfigs() {
+        return this.lbConfigs;
+    }
+
+    public void setLbConfigs(List<? extends LoadBalancerConfig> lbConfigs) {
+        if (lbConfigs == null) {
+            lbConfigs = new ArrayList<LoadBalancerConfig>();
+        }
+        this.lbConfigs = new LoadBalancerConfigTO[lbConfigs.size()];
```

Comment:
@weizhouapache advised changes to improve similar code, consider this as well. thanks.

Reply:
@sureshanaparti this has been changed in commit  as well 
https://github.com/apache/cloudstack/pull/4141/commits/1ccff8298166d66479d303b5be9196c23c8d0814